src/app/quiz/body.js

```diff
@@ -0,0 +1,75 @@
+'use client'
+
+import Box from "@mui/material/Box";
+import {useTheme} from '@mui/material/styles';
+import Container from "@mui/material/Container";
+import Grid from '@mui/material/Grid';
+
+function VariantButton() {
+  const theme = useTheme()
+  return (
+    <Box
+      sx={{
+        backgroundColor: theme.palette.quiz.button.backgroundColor,
+        padding: 3,
+        borderRadius: 0.6,
+        margin: 0.6,
+        textAlign: "center",
+      }}
+    >Variant 1</Box>
+  )
+}
+
+export default function Body() {
+  const theme = useTheme()
+  return (
+    <Box
+      component="main"
+      sx={{
+        backgroundColor: theme.palette.quiz.backgroundColor,
+        padding: 3,
+        alignItems: 'center',
+      }}
+    >
+      <Container maxWidth="lg" sx={{
+        display: "flex",
+        flexDirection: "column",
```

Comment:
It does not make sense to make a flex container here, as for me. 
You are putting child boxes one under another. That is exactly what layout by default does.
I'd remove 35-38.